hardhat.config.ts

```diff
@@ -26,16 +27,55 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
-// You need to export an object to set up your config
-// Go to https://hardhat.org/config/ to learn more
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+const config: HardhatUserConfig = {
+  solidity: "0.8.2",
+  networks: {
+    hardhat: {
+      live: false,
+      blockGasLimit: 6800000,
+      chainId: HARDHAT_TEST_NET_CHAIN_ID,
+      gasPrice: 60000000,
+      hardfork: 'istanbul', // London hardfork is incompatible with RSK gasPrice
 
-const ACCOUNTS = {
-  count: 1,
-  mnemonic: process.env.MNEMONIC,
-  path: "m/44'/60'/0'/0",
+      tags: ['test', 'local'],
+    },
+    // RSK
+    rsktestnet: {
+      live: true,
+      url: 'https://public-node.testnet.rsk.co',
+      blockGasLimit: 6800000,
+      gasPrice: 68000000, // 0.06 gwei
+      chainId: RSK_TEST_NET_CHAIN_ID,
+      hardfork: 'istanbul', // London hardfork is incompatible with RSK gasPrice
+      accounts: {
+        count: 1,
+        mnemonic: process.env.MNEMONIC,
+        path: "m/44'/60'/0'/0",
+      }
+    },
+  },
+  namedAccounts: {
+    deployer: 0,
+    minter: 1,
+    multiSig: getMultiSigAddressesByChainId(),
+    proxyAdmin: getProxyAdminAddressesByChainId(),
+    bridgeProxy: getBridgeProxyAddressesByChainId(),
```

Comment:
I think bridge proxy is never used by the swap and can be safetly removed

Reply:
done